test/server/rpc/jsdoc-extractor.spec.js

```diff
@@ -0,0 +1,30 @@
+const assert = require('assert'),
+    utils = require('../../assets/utils.js'),
+    jp = utils.reqSrc('rpc/jsdoc-extractor.js');
+
+describe('jsdoc-extractor', () => {
+
+    describe.only('fnFinder', () => {
+        let testText = `let doStuff = a => a*2;
+    function doStuff(){}
+    GoogleMap.doStuff = function
+    GoogleMap.doStuff = (asdf) =>`;
+
+        let testLines = testText.split('\n');
+
+        it('should support multiline', () => {
+            assert.deepEqual(jp._findFn(testLines[2]), 'doStuff');
+        });
+
+        it('should find let fn = ()', () => {
+            let line = 'let reverseGeocode = (lat, lon, response, query)=>{';
```

Comment:
what about an arrow function with only one argument and no parenthesis?

Reply:
I added that. let me know if there is any other cases